src/content/self-hosted/digitalocean.mdx

```diff
@@ -78,15 +78,13 @@ Download a copy of the [Okteto DigitalOcean configuration file](https://www.okte
 For example:
 
 ```yaml
-license: 1234567890ABCD==
```

Comment:
Why do we remove the value of the license?

Reply:
That's a fake value, it's better to leave it empty, no?

Reply:
no, since we have example values for all the other fields in the values.yaml, we should leave it. This is equally fake than using 'example.com' as the domain

Reply:
@rberrelleza updated